Assets/Content/Furniture/Machines/Atmospherics/Canister/Canister.cs

```diff
@@ -9,19 +9,34 @@
 using UnityEngine.Tilemaps;
 
 namespace SS3D.Content.Furniture.Machines.Atmospherics {
+    // This handles the gas canister process, where we can put only Atmos stuff.
+    // This can only store gas (for now), and we have a maximum ammount of gas that can be stored.
+    // You can open the valve to release gas and you can set up the pressure the gas leak
```

Comment:
"You can open the valve to release gas and you can also set the release pressure"
this is more clear.